plugins/inputs/postgresql_extensible/test.sql

```diff
@@ -0,0 +1 @@
+select * from pg_stat_database
```

Comment:
Put this into a `dev` or `testfiles` directory and reference it as such

Reply:
Name the directory `testdata`, since this is treated specially and is idiomatic:
```
$ go help packages | grep testdata -C1
Directory and file names that begin with "." or "_" are ignored
by the go tool, as are directories named "testdata".
```